src/toTransform.js

```diff
@@ -35,14 +35,50 @@
     return 'scale3d(' + valuesArray.join(', ') + ')';
   }
 
+  function convertPath (properties) {
+    var offsetPath = null;
+    if ('offset-path' in properties) {
+      offsetPath = internalScope.offsetPathParse(properties['offset-path']);
+    }
+
+    if (offsetPath === undefined || offsetPath === null || offsetPath.type !== 'path') {
+      return null;
+    }
+
+    var offsetDistance;
+    if (('offset-distance' in properties) && (properties['offset-distance'] !== undefined)) {
+      offsetDistance = internalScope.offsetDistanceParse(properties['offset-distance']);
+    }
+
+    if (offsetDistance === undefined) {
+      offsetDistance = {value: 0, unit: 'px'};
+    }
+
+    var pathElement = document.createElementNS('http://www.w3.org/2000/svg', 'path');
```

Comment:
This will create a path element (and possibly more) on every animation frame. This has terribad performance implications and should have a loud TODO to recycle these elements.